extensions/javascript/schemas/jsconfig.schema.json

```diff
@@ -9,42 +9,132 @@
 						"umd",
+						"amd",
 						"system",
 						"es6",
 						"es2015"
 					]
 				},
-
+				"newLine": {
+					"description": "When down-level compiling, specifies the end of line sequence to be used when emitting files: 'CRLF' (dos) or 'LF' (unix).",
+					"enum": [
+						"CRLF",
+						"LF"
+					]
+				},
+				"noEmit": {
+					"description": "When down-level compiling, do not emit output.",
+					"type": "boolean"
+				},
+				"noEmitHelpers": {
+					"description": "When down-level compiling, do not generate custom helper functions like __extends in compiled output.",
+					"type": "boolean"
+				},
+				"noEmitOnError": {
+					"description": "When down-level compiling, do not emit outputs if any type checking errors were reported.",
+					"type": "boolean"
+				},
 				"noLib": {
 					"description": "Do not include the default library file (lib.d.ts).",
 					"type": "boolean"
 				},
+				"noResolve": {
+					"description": "When down-level compiling, do not resolve triple-slash references or module import targets to the input files.",
+					"type": "boolean"
+				},
+				"skipDefaultLibCheck": {
+					"description": "Do not check for the default library (lib.d.ts).",
+					"type": "boolean"
+				},
+				"outFile": {
+					"description": "When down-level compiling, concatenate and emit output to single file.",
+					"type": "string",
+					"format": "uri"
+				},
+				"outDir": {
+					"description": "When down-level compiling, redirect output structure to the directory.",
+					"type": "string",
+					"format": "uri"
+				},
+				"pretty": {
+					"description": "When down-level compiling, stylize errors and messages using color and context. (experimental)",
+					"type": "boolean"
+				},
+				"removeComments": {
+					"description": "When down-level compiling, do not emit comments to output.",
+					"type": "boolean"
+				},
+				"rootDir": {
+					"description": "When down-level compiling, specifies the root directory of input files. Use to control the output directory structure with --outDir.",
+					"type": "string",
+					"format": "uri"
+				},
+				"isolatedModules": {
+					"description": "When down-level compiling, unconditionally emit imports for unresolved files.",
+					"type": "boolean"
+				},
+				"sourceMap": {
+					"description": "When down-level compiling, generates corresponding '.map' file.",
+					"type": "boolean"
+				},
+				"sourceRoot": {
+					"description": "When down-level compiling, specifies the location where debugger should locate JavaScript files instead of source locations.",
+					"type": "string",
+					"format": "uri"
+				},
+				"stripInternal": {
+					"description": "When down-level compiling, do not emit declarations for code that has an '@internal' annotation.",
+					"type": "boolean"
+				},
 				"target": {
-					"description": "Specify ECMAScript target version.",
+					"description": "Specifies the default libarary used and, when down-level compiling, what code is generated.",
```

Comment:
Actually this has an option on intellisense. Suggest:
Specify ECMAScript target version and the default library .d.ts to use.